wazo_webhookd/services/http/plugin.py

```diff
@@ -70,19 +96,17 @@ def run(
 
         if body:
             content_type = options.get('content_type', 'text/plain')
-            # NOTE(sileht): parse_header will drop any erroneous options
```

Comment:
@fblackburn1 looking at the cgi.parse_header code, this seems false.
```
def parse_header(line):
    """Parse a Content-type like header.

    Return the main content-type and a dictionary of options.

    """
    parts = _parseparam(';' + line)
    key = parts.__next__()
    pdict = {}
    for p in parts:
        i = p.find('=')
        if i >= 0:
            name = p[:i].strip().lower()
            value = p[i+1:].strip()
            if len(value) >= 2 and value[0] == value[-1] == '"':
                value = value[1:-1]
                value = value.replace('\\\\', '\\').replace('\\"', '"')
            pdict[name] = value
    return key, pdict
```
So I just removed the comment.
We *could* filter out options not defined for the content-type header spec.